benchopt/cli/main.py

```diff
@@ -104,6 +116,10 @@ def run(benchmark, solver_names, forced_solvers, dataset_names,
             max_runs=max_runs, n_repetitions=n_repetitions,
             timeout=timeout, plot_result=plot, html=html, pdb=pdb
         )
+
+        if do_profile:
+            print_stats()
```

Comment:
You can also simply do as it will only print stat if they have been recorded.
```suggestion
        print_stats()
```